students/AndrewMiotke/lesson01/assignment/inventory_management/main.py

```diff
@@ -0,0 +1,99 @@
+""" Launches the user interface for the inventory management system """
+import sys
+import market_prices
```

Comment:
These imports need to be modified so that inventory_management is added to the beginning to fix relative imports